docs: langflow assistant feature (#12439)
langflow-ai/langflow: docs/docs/Support/release-notes.mdx

    Tool calling is enabled for the following Google Gemini 3 preview models:
    `gemini-3.1-pro-preview`, `gemini-3-pro-preview`, `gemini-3-flash-preview`, and `gemini-3-pro-image-preview`.

- Build flows and components with **Langflow Assistant**

>     **Langflow Assistant** is a graph-aware co-pilot that can create flows and components from natural language prompts, directly within the Langflow workspace.

Reviewer: Not sure about the term graph-aware. This could be mistaken with graph traversals and vector search contexts.

```suggestion
    **Langflow Assistant** is a built-in assistant that can help you create flows and custom components from natural language prompts.
```
